### PR TITLE
Add Vercel Deployment Project Tutorial 

### PR DESCRIPTION
**Pull request by exrlla**

### CODE REVIEW

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+<img src="https://raw.githubusercontent.com/codedex-io/projects/main/projects/deploy-a-website-with-vercel/images/vercel-congrats.png" alt="the vercel dashboard with confetti and a congratulations screen" />
+
+You’re now able to access my linktree clone website at [bento-blond.vercel.app](https://bento-blond.vercel.app/). 
+
+You'll notice your GitHub repository now looks a bit different! Vercel interacted with your GitHub repository to automate the deployment process and ensure that your website stays up to date with the latest changes in your codebase. That means everytime you push code to the main branch (or the branch specified in Vercel), so you don't have to worry about updating it yourself everytime! You have successfully deployed your website! 
+
+## Conclusion
+
+Congratulations! 🎆
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:101]:
```suggestion
Congratulations! 🎊 
```

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+You have successfully deployed your static website using Vercel and GitHub! Now, you’re able to expand to using your own domain if you choose, or creating more websites to share with friends! 👯
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:103]:
```suggestion
You have successfully deployed your static website using Vercel and GitHub! Now, you can use your own domain if or create more websites to share with friends! 👯
```

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+uid: 11zhRKeJCWWcD7IkTJBtuK3Mkvo1
+datePublished: YYYY-MM-DD
+description: Learn how to deploy a static website using Vercel.
+header:
+bannerImage:
+tags:
+    - javascript
+    - html
+    - css
+    - intermediate
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:13]:
```suggestion
  - javascript
  - html
  - css
  - intermediate
```

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
+title: Deploy A Website with Vercel
+author: Ellie Popoca
+uid: 11zhRKeJCWWcD7IkTJBtuK3Mkvo1
+datePublished: YYYY-MM-DD
+description: Learn how to deploy a static website using Vercel.
+header:
+bannerImage:
+tags:
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:9]:
```suggestion
bannerImage:
published: false
tags:
```

The `published` property will need to be here (and ultimately changed to `true` by @Goku-kun once it's ready to go live)

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx (hunk 3 above)
Comment (Dusch4593) [reply projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:13]: Just 2 spaces

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@exrlla Just left some comments/suggestions for you!

Thank you for sharing this PR with us! Super excited for this one. 🚀

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+### Getting Started with Vercel
+
+Vercel lets us deploy a static website that consists of fixed content and doesn't require server-side processing or a dynamic server to generate web pages. Instead, all the content of a static website is pre-built and stored as HTML, CSS, JavaScript, images, and other assets.
+
+This tutorial works with code that is pushed to a repository (repo) on [GitHub](https://github.com/).
+
+Before we begin, let's do the following:
+ 
+1. Create and/or log on to your GItHub account.
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:57]:
```suggestion
1. Create and/or log on to your GitHub account.
```

Comment (Dusch4593) [reply projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:57]: This was probably of my doing 😂

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+2. Ensure the code you would like to deploy with Vercel is all saved in a folder and pushed to a repo on GitHub.
+
+**Note:** If at this point you don’t have your project on a GitHub repository, [follow these instructions](https://docs.github.com/en/migrations/importing-source-code/using-the-command-line-to-import-source-code/adding-locally-hosted-code-to-github) on adding locally hosted code to GitHub.
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:60]:
```suggestion
**Note:** If you aren't sure of how to add your project to a GitHub repository, [follow these instructions](https://docs.github.com/en/migrations/importing-source-code/using-the-command-line-to-import-source-code/adding-locally-hosted-code-to-github) on adding locally hosted code to GitHub.
```

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+Once you have authenticated your GitHub account, the screen should now look like this:
+
+<img src="https://raw.githubusercontent.com/codedex-io/projects/main/projects/deploy-a-website-with-vercel/images/vercel-authed.png" alt="The Vercel dashboard with Ellie's recent repositories shown in a list with an option to import to vercel" />
+
+On the Vercel dashboard, you may notice some repositories that were recently committed to! ✨
+
+### Deploying Your Website
+
+At this point, depending on your project, it should be able to display and work locally on your computer Below is a linktr.ee clone that I created!
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:74]:
```suggestion
At this point, depending on your project, it should be able to display and work locally on your computer. Below is a linktr.ee clone that I created!
```

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+You’ll also notice some other settings before deploying such as:
+
+- Overrides for the "Build and Output Settings."
+- Settings for "Environment Variables" such as secrets, keys, and passwords.
+
+Since we’re deploying a static site, we won’t need to change this information. 
+
+**Note:** This static website was created with Next.js, a React-based web framework. However, you can change the framework that you’re using for deployment. Vercel should detect this automatically, but here are some of the options that you will see under "Framework Preset": 
+
+<img src="https://raw.githubusercontent.com/codedex-io/projects/main/projects/deploy-a-website-with-vercel/images/vercel-fw-demo.gif" alt="GIF with the vercel dashboard showing a list of popular frameworks" />
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:95]:
```suggestion
<img src="https://raw.githubusercontent.com/codedex-io/projects/main/projects/deploy-a-website-with-vercel/images/vercel-fw-demo.gif" alt="GIF with the Vercel dashboard showing a list of popular frameworks" />
```

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+Once you’re all set, you can now click "Deploy"! You’ll now notice the build process starting. 
+
+<img src="https://raw.githubusercontent.com/codedex-io/projects/main/projects/deploy-a-website-with-vercel/images/vercel-loading.png" alt="the vercel dashboard with an yellow arrow and a box around the text that shows the loading screen for deployment" />
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:99]:
```suggestion
<img src="https://raw.githubusercontent.com/codedex-io/projects/main/projects/deploy-a-website-with-vercel/images/vercel-loading.png" alt="The Vercel dashboard with a yellow arrow and a box around the text that shows the loading screen for deployment" />
```

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+This part can take a couple of minutes! Once the deployment is finished, you should see the congratulations screen!! 🎉
+
+<img src="https://raw.githubusercontent.com/codedex-io/projects/main/projects/deploy-a-website-with-vercel/images/vercel-congrats.png" alt="the vercel dashboard with confetti and a congratulations screen" />
```
Comment (Dusch4593) [inline projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:103]:
```suggestion
<img src="https://raw.githubusercontent.com/codedex-io/projects/main/projects/deploy-a-website-with-vercel/images/vercel-congrats.png" alt="The Vercel dashboard with confetti and a congratulations screen" />
```

File: projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+You’re now able to access my linktree clone website at [bento-blond.vercel.app](https://bento-blond.vercel.app/). 
+
+You'll notice your GitHub repository now looks a bit different! Vercel interacted with your GitHub repository to automate the deployment process and ensure that your website stays up to date with the latest changes in your codebase. That means everytime you push code to the main branch (or the branch specified in Vercel), so you don't have to worry about updating it yourself everytime! You have successfully deployed your website!
```
[the remark this answers is not in the document]
Comment (Dusch4593) [reply projects/deploy-a-website-with-vercel/deploy-a-website-with-vercel.mdx:107]: Wasn't an image of the green check on GitHub somewhere around here in the draft? This image, right?

projects/deploy-a-website-with-vercel/images/Screenshot 2023-08-01 at 8.16.00 AM.png

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@exrlla Thanks for making those changes! We are super close to getting this published. I went through once more and have some more points of feedback. Could we take a look at these comments/suggestions?

**Comment by sonnynomnom** [comment]
Late the to game but let's freaking goooooo! 🚀

**Pull request merged**